administrator/language/en-GB/en-GB.lib_joomla.ini

```
@@ -361,9 +361,9 @@ JLIB_HTML_BATCH_FLIPORDERING_LABEL="Reverse the ordering of all articles in the
 JLIB_HTML_BATCH_LANGUAGE_LABEL="Set Language"
 JLIB_HTML_BATCH_LANGUAGE_LABEL_DESC="Not making a selection will keep the original language when processing."
 JLIB_HTML_BATCH_LANGUAGE_NOCHANGE="- Keep original Language -"
-JLIB_HTML_BATCH_MENU_LABEL="To Move or Copy your selection please select a Category."
+JLIB_HTML_BATCH_MENU_LABEL="Select a Category for your Selection"
 JLIB_HTML_BATCH_MOVE="Move"
-JLIB_HTML_BATCH_MOVE_QUESTION="Do you want to move the items or make a copy of them?"
+JLIB_HTML_BATCH_MOVE_QUESTION="Copy or Move the items?"
```

```suggestion
JLIB_HTML_BATCH_MOVE_QUESTION="Move or Copy the items?"
```
To be consistent with the next language string.

---

Given the copy option shows before the move option in the UI probably the correct thing is to change the string below too